client/src/app/domain/models/mediafiles/mediafile.ts

```diff
@@ -38,16 +38,46 @@ export class Mediafile extends BaseModel<Mediafile> {
         super(Mediafile.COLLECTION, input);
     }
 
-    public used_as_logo_in_meeting_id(place: string): Id | null {
+    /**
+     * Check if the image is used at the given place. If place is empty, checks
+     * if the image is used in any place and returns the first meeting id found.
+     *
+     * @param place The places the image can be used
+     * @returns the meeting id or `null` if the image is not used.
+     */
+    public used_as_logo_in_meeting_id(place?: string): Id | null {
+        if (!place) {
+            return this.used_in_meeting(this.used_as_logo_$_in_meeting_id, 'logo');
+        }
         const path = `used_as_logo_$${place}_in_meeting_id` as keyof Mediafile;
         return (this[path] as Id) || null;
     }
 
-    public used_as_font_in_meeting_id(place: string): Id | null {
+    /**
+     * Check if the font is used at the given place. If place is empty, checks
+     * if the font is used in any place and returns the first meeting id found.
+     *
+     * @param place The text parts the font can be used
+     * @returns the meeting id or `null` if the font is not used.
+     */
+    public used_as_font_in_meeting_id(place?: string): Id | null {
+        if (!place) {
+            return this.used_in_meeting(this.used_as_font_$_in_meeting_id, 'font');
+        }
         const path = `used_as_font_$${place}_in_meeting_id` as keyof Mediafile;
         return (this[path] as Id) || null;
     }
 
+    private used_in_meeting(list: string[], type: string): Id | null {
```

Comment:
I think this could still easily handle the rest of the original functionalities.

Also if you're already creating the path explicitly, you might as well get the list within this function by using the bracket notation (i.e. by calling `this['used_as_${type}_$_in_meeting_id']` inside the method). 
That way you don't have to specifically ask for the list as a parameter.

Reply:
For sure this is possible! I thought of these function a bit more type specific like 'isFont' or something. Therefore I left the original function names and the code duplication.
To resolve this like you stated means the type has to be a parameter. Would that be better?

Reply:
I think whether or not you should keep the original function names depends on how you feel about changing all the function calls, but the code duplication should probably be built out anyway.